packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -9,18 +9,18 @@ export interface LambdaRuntimeProps {
 /**
  * Lambda function runtime environment.
  */
-export class LambdaRuntime {
+export class LambdaRuntime implements InlinableLambdaRuntime, InlinableJavascriptLambdaRuntime {
     public static readonly NodeJS = new LambdaRuntime('nodejs', { supportsInlineCode: true }) as InlinableJavascriptLambdaRuntime;
-    // Using ``as InlinableLambdaRuntime`` because that calss cannot be defined just yet
+    // Using ``as InlinableLambdaRuntime`` because that class cannot be defined just yet
```

Comment:
Don't know if this still matters since we should probably just get rid of the interfaces, but if this crops up again, the way to solve it is thusly:

```
public static get NodeJS43(): InlinableJavascriptLambda {
    return new LambdaRuntime(...);
}
```

I.e., use a getter to temporally decouple type declaration from returned type. Types *can* use forward references, instantiations *can't* (because JavaScript).

Reply:
Oh I see, `InlinableJavascriptLambda` was probably intended to be a class, but because of the issue of getting it to declare properly we made it an interface?

Reply:
So intended was probably:

```ts
public static get NodeJS43(): LambdaRuntime {
    return new InlinableJavascriptLambda(...);
}
```

But anyway, a concrete `LambdaRuntime` class without interfaces probably suffices.